.changes/unreleased/charts-redpanda-Fixed-20250418-150810.yaml

```diff
@@ -0,0 +1,4 @@
+project: charts/redpanda
+kind: Fixed
+body: Respect internal Kafka listener trustStore configuration when schema_registry_client and pandaproxy_client kafka client configuration is generated.
```

Comment:
This would be much more clear if it referenced field names from our `values.yaml`. A link back to the original issue might help as well. How about:

```suggestion
body: The `schema_registry_client` and `pandaproxy_client` stanzas of `redpanda.yaml` 
 now respect `listeners.kafka.tls.trustStore`, when provided. See also [#123](link to issue).
```